<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrialSettingsDialog</class>
 <widget class="QDialog" name="TrialSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Trial Settings</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter information for one game, then click 'add' to add game to trial. When done adding games, click 'Start'</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="gameNumberLabel">
     <property name="text">
      <string>Participant ID</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="pidEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ageLabel">
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="ageEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="genderLabel">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="genderComboBox">
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="inputLabel">
     <property name="text">
      <string>Interface</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="inputComboBox">
     <item>
      <property name="text">
       <string>Leap Controller</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keyboard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mouse</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Games</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton_2">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton_2">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="1">
    <widget class="QTableView" name="gameTableView">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="midLineWidth">
      <number>9</number>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton_2">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="colorLabel">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="soundLabel">
     <property name="text">
      <string>Sound</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="soundComboBox">
     <item>
      <property name="text">
       <string>3D (Surround)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Sound</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stereo</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="colorComboBox">
     <item>
      <property name="text">
       <string>Color</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grayscale</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Color</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
